### PR TITLE
radio changes and update

### PR DESCRIPTION
**Pull request by grayherm**

### CODE REVIEW

File: docs/3-advanced-techniques/17-advanced-radio-procedures.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 
-The goal is to allow you to get a quick understanding of what we expect regarding the use of radio and how you should communicate with your fellow members. Good communication is a very important part of any operation as it will greatly improve the experience of everyone involved.
+This should supply you with a quick understanding of what we expect regarding the use of radio and how you should communicate with your fellow members. Good communication is a **very** important part of any operation as it will greatly improve the experience of everyone involved.
 
 ## 2. Radio Protocols
 
 1. Make sure that your message is **clear**.
-2. Keep messages as **short and sweet** as possible.
+2. Keep messages as **short and concise** as possible.
 3. Always acknowledge that you **recieved and understood** the order given.
```
Comment (MrFlufficans) [inline docs/3-advanced-techniques/17-advanced-radio-procedures.md:13]: I before E except after C <3

File: docs/3-advanced-techniques/17-advanced-radio-procedures.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
-4. If your last message had **wrong information**, **correct it immediately** and **send** again.
+4. If your last message had the **wrong information**, **correct it immediately** and **send** again.
```
Comment (MrFlufficans) [inline docs/3-advanced-techniques/17-advanced-radio-procedures.md:14]: If we want to be fancy I would say "incorrect information"

File: docs/3-advanced-techniques/17-advanced-radio-procedures.md [hunk 3]
```diff
@@ -27,27 +27,27 @@ The goal is to allow you to get a quick understanding of what we expect regardin
...
-  - 1 for radio channel to contact the artilery element (hammer)
+  - 1 for radio channel to contact Command Team (Crossroads).
+  - 1 for radio channel to contact the section elements (wolfpack).
+  - 1 for radio channel to contact the artilery element (hammer).
 
-More radios may be required to cover all the channels depending on the elements present during an operation. The use of a 117 may be required for elements operating farther away.
+More radios may be required to cover all channels depending on the elements present during an operation. The use of a 117 may be required for elements operating farther away.
 
-During an operation, you will have to switch your PTT between the different radios to talk on the different radio channels (you will always hear from all the radios even those you are not speaking into). You can for example set your radios as follow :
+During an operation, you will have to switch your push to talk between the different radios in order talk on each individual radio channels (you will always hear from all the radios even those you are not speaking into). You can for example set your radios as follow :
```
Comment (MrFlufficans) [inline docs/3-advanced-techniques/17-advanced-radio-procedures.md:44]: missing a "to" I believe after order 

*radios in order to talk on each individual radio channels*

**Review by MrFlufficans — CHANGES_REQUESTED** [review]
Seems solid to me, has very few "errors"

**Review by Luigikissme — APPROVED** [review]
im not good at spelling so thats on you but the info seems to be right to my knowledge

**Review by MrFlufficans — APPROVED** [review]
Corrected all the problems I mentioned, Looks good to me

**Review by zuedev — APPROVED** [review]

**Pull request merged**